client/dashboard/emails/dataviews.tsx

```diff
@@ -3,12 +3,23 @@ import {
 	__experimentalHStack as HStack,
 	__experimentalVStack as VStack,
 	Icon,
+	Button,
 } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
 import { __, sprintf } from '@wordpress/i18n';
 import { next, wordpress } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from 'react';
 import { purchasesRoute } from '../app/router/me';
 import { Text } from '../components/text';
 import GoogleLogo from './resources/google-logo';
+import MailboxIcon from './resources/mailbox-icon';
+import {
+	buildGoogleFinishSetupLink,
+	buildGoogleMailboxLink,
+	buildGoogleManageWorkspaceLink,
+	buildTitanMailboxLink,
+} from './utils/email-utils';
 import type { Action, Field, View } from '@wordpress/dataviews';
 
 // Shared fields for Emails DataViews
```

Comment:
This file is pretty big now, I'm considering splitting it up, but not in this PR to avoid noise and refactors here.